confignode/src/main/java/org/apache/iotdb/confignode/persistence/partition/PartitionInfo.java

```diff
@@ -149,7 +149,7 @@ public void addMetrics() {
   }
 
   public int generateNextRegionGroupId() {
```

Comment:
"nextRegionGroupId.set(-1 )"  in  "clear()" method.

Reply:
fixed~